test-requirements.txt

```diff
@@ -1,2 +1,3 @@
 pytest
 pytest-cov
+hypothesis
```

Comment:
shouldn't we add a >=3.64.0 rule to this requirement ?

Reply:
Sure!  You should probably add minimum versions to pytest and pytest-cov too then, as I'm sure there are some old enough that they don't work...